packages/react/src/components/ProgressBar/ProgressBar.jsx

```diff
@@ -99,6 +103,72 @@ const ProgressBar = ({
   const fillColor = hasColorObject ? matchingThreshold.color.fill : matchingFillColor;
   const strokeColor = hasColorObject ? matchingThreshold.color.stroke : undefined;
 
+  /**
+   * helper to render the value and unit
+   *
+   * @returns JSXElement
+   */
+  const renderValueAndUnit = () => (
+    <span
+      className={classnames(`${iotPrefix}--progress-bar__value-label`, {
+        // allow styling the value label differently when above max
+        [`${iotPrefix}--progress-bar__value-label--over`]: value > max,
+        [`${iotPrefix}--progress-bar__value-label--inline`]: inline,
+      })}
+    >{`${value}${valueUnit}`}</span>
+  );
+
+  /**
+   * When the progress bar is not using the inline variant, render the label and icon
+   * above the progress bar.
+   *
+   * @returns JSX Element
+   */
+  const renderProgressBarLabel = () => (
+    <div
+      className={classnames(`${iotPrefix}--progress-bar__label--right`, {
+        [`${prefix}--visually-hidden`]: hideLabel,
+      })}
+    >
+      {Icon ? (
+        <span className={`${iotPrefix}--progress-bar__icon`} data-testid="progress-bar-icon">
+          {renderIconByName && typeof Icon === 'string' ? (
+            renderIconByName(Icon, {
+              fill: fillColor,
+              stroke: strokeColor,
+              'aria-label': mergedI18n.iconLabel,
+            })
+          ) : (
+            <Icon fill={fillColor} stroke={strokeColor} aria-label={mergedI18n.iconLabel} />
+          )}
+        </span>
+      ) : null}
+      {renderValueAndUnit()}
+    </div>
+  );
+
+  /**
+   * helper method to render the progress bar with a div wrapper for the inline variant
+   *
+   * @returns JSXElement
+   */
+  const renderProgressBarWithInlineWrapper = () => (
+    <div
+      className={classnames({
+        [`${iotPrefix}--progress-bar-wrapper--inline`]: inline,
+      })}
+    >
+      <CarbonProgressBar
+        label={label}
+        helperText={!inline ? helperText : undefined}
+        hideLabel={hideLabel || inline}
```

Comment:
Same here, we can just always hide the label

Reply:
Fixed.